mage_ai/frontend/components/PipelineDetail/KernelStatus.tsx

```diff
@@ -19,12 +19,14 @@ import Tooltip from '@oracle/components/Tooltip';
 import dark from '@oracle/styles/themes/dark';
 import { Close, FileFill } from '@oracle/icons';
 import { FileTabStyle, PipelineHeaderStyle } from './index.style';
+import { KEY_CODE_ENTER, KEY_CODE_META } from '@utils/hooks/keyboardShortcuts/constants';
 import { ThemeType } from '@oracle/styles/themes/constants';
 import { PADDING_UNITS, UNIT } from '@oracle/styles/units/spacing';
 import { dateFormatLongFromUnixTimestamp } from '@utils/string';
 import { goToWithQuery } from '@utils/routing';
 import { remove } from '@utils/array';
 import { pauseEvent } from '@utils/events';
+import { useKeyboardContext } from '@context/Keyboard';
```

Comment:
do you use these imports?

Reply:
oh I'll remove, I was working on keyboard shortcuts but ran into some super annoying bugs. I'll open those changes as a draft PR if someone wants to take a look/reimplement later